helper-cli/src/main/kotlin/commands/SubtractScanResultsCommand.kt

```diff
@@ -41,23 +41,23 @@ internal class SubtractScanResultsCommand : CliktCommand(
 ) {
     private val lhsOrtFile by option(
         "--lhs-ort-file",
-        help = "The ORT result containing the left-hand-side scan result."
+        help = "The ORT result containing the left-hand side scan result."
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
         .convert { it.absoluteFile.normalize() }
         .required()
 
     private val rhsOrtFile by option(
         "--rhs-ort-file",
-        help = "The ORT result containing the left-hand-side scan result."
+        help = "The ORT result containing the right-hand side scan result."
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
         .convert { it.absoluteFile.normalize() }
         .required()
 
     private val outputOrtFile by option(
         "--output-ort-file",
-        help = "The ORT result containing the left-hand-side scan result."
+        help = "The output ORT result."
```

Comment:
This sounds a bit as if the output is already there. How about "The file to write the output ORT result to."?